cli/src/main/scala/com/lightbend/rp/reactivecli/package.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 Lightbend, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.rp
+
+package object reactivecli {
+  def someIf[T](b: Boolean)(v: => T): Option[T] =
```

Comment:
They're semantically different - `v` is call-by-name so it will only be invoked if `b` is true. Normally I'd use scalaz's `Boolean.option` (i.e. `true.option(...)`) method but we're not using that.